Core/include/Acts/Seeding/Seedfinder.hpp

```diff
@@ -21,32 +24,33 @@
 #include <utility>
 #include <vector>
 
-namespace Acts {
-struct LinCircle {
-  float Zo;
-  float cotTheta;
-  float iDeltaR;
-  float Er;
-  float U;
-  float V;
-};
-template <typename external_spacepoint_t>
+#ifdef  ACTS_HAS_CUDA
+#include <Acts/Seeding/SeedfinderCudaKernels.cuh>
+#endif
```

Comment:
If this was in a separate CUDA target, which is only compiled when the flag is set, this preprocessor guard wasn't required I think.